tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -646,3 +646,8 @@ test_i2c_as5600: as5600@64 {
 	compatible = "ams,as5600";
 	reg = <0x64>;
 };
+
+test_i2c_htu21d: htu21d@40 {
```

Comment:
Increment the address by one from the last entry. This is a build test that will never run on hardware, so the address doesn't need to match the datasheet.